cmd/optimizely/main.go

```diff
@@ -286,7 +290,7 @@ func main() {
 
 	log.Info().Str("version", conf.Version).Msg("Starting services.")
 	sg.GoListenAndServe("api", conf.API.Port, apiRouter)
-	sg.GoListenAndServe("webhook", conf.Webhook.Port, routers.NewWebhookRouter(optlyCache, conf.Webhook))
+	sg.GoListenAndServe("webhook", conf.Webhook.Port, routers.NewWebhookRouter(ctx, optlyCache, *conf))
```

Comment:
If we send pointers, we'll need an additional nil pointer check every time. Just want to avoid this here. conf values won't change inside NewWebhookRouter.